assets/src/edit-story/components/panels/videoAccessibility.js

```diff
@@ -89,6 +104,7 @@ function VideoAccessibilityPanel({ selectedElements, pushUpdate }) {
           onChange={(value) => pushUpdate({ alt: value || null })}
```

Comment:
^ There's title as well, probably should also have a limit?

Reply:
@o-fernandez should we put it [here](https://docs.google.com/spreadsheets/d/1hdru-_p4Yk-VFQ1rEFGYHHI2A7eNZkxLA33-LblG8sE/edit#gid=0)?

Reply:
Story title limit? I think we can use: 
min = 0
max = 75 characters

I'll update the spreadsheet to include. It's a reasonable limit. One thing, though, is to ensure we count characters properly for non-standard/roman characters (e.g., accented characters, foreign scripts like chinese/japanese characters, etc.)

Reply:
This is for the video title attribute I think. Similar to the alt attribute / assistive text.

Reply:
Looking up online, there doesn't seem to be browser imposed limits on "title" attribute, except some places mention 512 or 1024 characters limit on some browsers (IE/Chrome)

https://stackoverflow.com/questions/8516235/max-length-of-title-attribute#:~:text=6%20Answers&text=Titles%20are%20limited%20to%20512,internet%20explorer%20according%20to%20MSDN.&text=I%20just%20tried%20this%20out,Chrome%20limits%20to%201024. 

Should we go with 512 for all title fields? Not sure if there's a WordPress standard we can just follow. @swissspidy

Reply:
@o-fernandez WP doesn't seem to set a limit for title attribute for media, other than the DB limit for `text` type.

Reply:
512 or no limit? WDYT @swissspidy

Reply:
Maybe we can go with 512 for now to unblock this PR and change it later in a follow-up PR if it comes out there's a better option. That should be almost the only blocker currently remaining. Thoughts?

Reply:
Or we can proceed without the limit.
> 512 or 1024 characters limit on some browsers (IE/Chrome)

IE is not supported and I think that Chrome limit is not valid anymore (just used a link with 6xxx chars).